paimon-api/src/main/java/org/apache/paimon/Snapshot.java

```diff
@@ -195,6 +196,13 @@ public class Snapshot implements Serializable {
     @Nullable
     protected final String statistics;
 
+    // properties
+    // null for paimon <= 1.1
```

Comment:
What about empty? We don't want to include empty map.

Reply:
Now, I modified it to be null when empty map